master/internal/authz/obfuscate.go

```diff
@@ -13,6 +15,9 @@ import (
 const (
 	hiddenString = "********"
 	hiddenInt    = -1
+
+	// Used publicly for testing purposes.
+	HiddenString = hiddenString
```

Comment:
We can probably just make these public and drop the private version

Reply:
Gotchya, fixed!